### ISSUE TITLE
[2.3.5-p2 only]Upgrading to 2.3.5 breaks SOAP

### ISSUE THREAD

**Issue by steved1982** [magento/magento2#28255]
### Preconditions (*)

1. Magento 2.3.5-p2 CE

### Steps to reproduce (*)

1. Using a fresh install of Magento 2.3.5-p2
2. Try calling http://DOMAIN.com/soap/default?wsdl_list=1

### Expected result (*)

1. A list of all available services in WSDL

### Actual result (*)

1. ![image](https://user-images.githubusercontent.com/14214911/82200982-97d6c400-98f7-11ea-99f9-7bca9a71fea4.png)
2. This was working in Magento 2.3.4
See error: [default_error.zip](https://github.com/magento/magento2/files/5150550/default_error.zip)

**Comment by steved1982** [comment]
@magento give me 2.3.5 instance

**Comment by magento-engcom-team** [comment]
Hi @steved1982. Thank you for your request. I'm working on Magento 2.3.5 instance for you

**Comment by moloughlin** [comment]
Linking to resolution from my ticket (same root cause): https://github.com/magento/magento2/issues/28244#issuecomment-630077997

TL;DR: upgrade dotmailer module or use a replace directive to remove it.

**Comment by steved1982** [comment]
I have followed those steps but I am still seeing the error. My example URL is:
soap/all?wsdl&services=customerAccountManagementV1

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Delta 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-37231` were created

**Issue Available**: @engcom-Delta, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by sdzhepa** [comment]
The related internal Jira ticket `MC-37231` was closed as `non-reproducible` in `2.4-develop`.
It means that Magento team either unable to reproduce this issue using provided _Steps to Reproduce_ from the **_Description section_** on clean Magento instance or the issue has been already fixed in the scope of other tasks.

But if you still run into this problem on the latest `2.4-develop` please create new Issue with updated information/steps/preconditions in the **_Description section_**.
